fastparquet/api.py

```diff
@@ -90,17 +105,15 @@ def _read_partitions(self):
                     cats.setdefault(key, set()).add(val)
         self.cats = {key: list(v) for key, v in cats.items()}
 
-    def read_row_group_file(self, rg, columns, categories, filters={}):
+    def read_row_group_file(self, rg, columns, categories):
+        """Open file for reading, and process it as a row-group"""
         ofname = self.sep.join([os.path.dirname(self.fn),
                                 rg.columns[0].file_path])
         with self.open(ofname, 'rb') as f:
-            return self.read_row_group(rg, columns, categories,
-                                       filters={}, infile=f)
+            return self.read_row_group(rg, columns, categories, infile=f)
 
-    def read_row_group(self, rg, columns, categories, filters={},
-                       infile=None):
-        """Filter syntax: [(column, op, val), ...],
-        where op is [==, >, >=, <, <=, !=, in, not in]
+    def read_row_group(self, rg, columns, categories, infile=None):
+        """Access row-group in a file and read some columns into a data-frame.
```

Comment:
Style nit, request the following change for symmetry:

```
""" some text
"""
```

to either 

```
""" some text """
```

or 

```
""" 
some text 
"""
```